docs/generated/go.md

```diff
@@ -0,0 +1,201 @@
+---
+bodyclass: docs
+headline: Go Generated Code Guide
+layout: docs
+sidenav: doc-side-quickstart-nav.html
```

Comment:
You actually probably don't want the quickstart sidenav as otherwise your reference doc will end up with the quickstarts. :)

What we probably want is a nice new doc-side-reference-nav.html - at the moment we don't have one because we only have a single page in the reference section linking to the autogenerated docs, but if we're going to start to have more manually added reference pages we'll need one for users to navigate through the section.

Let me have a think about how to incorporate the existing reference landing page into a structure with a side nav and I'll get back to you...